dfastmi/gui/dialog_view.py

```diff
@@ -390,81 +512,76 @@ def activate_dialog(self) -> None:
         """
         self._win.show()
         sys.exit(self._app.exec_())
-
-
     
-    def menu_load_configuration(self) -> None:
+    def _menu_load_configuration(self) -> None:
         """
         Ask for a configuration file name and update GUI based on its content.
 
         Arguments
         ---------
         None
         """
-        fil = QtWidgets.QFileDialog.getOpenFileName(
-            caption=self.view_model.gui_text("select_cfg_file"), filter="Config Files (*.cfg)"
+        fil = QFileDialog.getOpenFileName(
+            caption=self._view_model.gui_text("select_cfg_file"), filter="Config Files (*.cfg)"
         )
         filename = fil[0]
         if filename != "":
-            if not self.view_model.load_configuration(filename):
-                self.showError(self.view_model.gui_text("file_not_found", prefix="", dict={"name": filename}))
+            if not self._view_model.load_configuration(filename):
+                self._showError(self._view_model.gui_text("file_not_found", prefix="", dict={"name": filename}))
 
-    def menu_save_configuration(self) -> None:
+    def _menu_save_configuration(self) -> None:
         """
         Ask for a configuration file name and save GUI selection to that file.
 
         Arguments
         ---------
         None
         """
-        fil = QtWidgets.QFileDialog.getSaveFileName(
-            caption=self.view_model.gui_text("save_cfg_as"), filter="Config Files (*.cfg)"
+        fil = QFileDialog.getSaveFileName(
+            caption=self._view_model.gui_text("save_cfg_as"), filter="Config Files (*.cfg)"
         )
         filename = fil[0]
         if filename != "":
-            self.view_model.save_configuration(filename)
-
+            self._view_model.save_configuration(filename)
 
-    def menu_about_self(self) -> None:
+    def _menu_about_self(self) -> None:
         """
         Show the about dialog for D-FAST Morphological Impact.
 
         Arguments
         ---------
         None
         """
-        msg = QtWidgets.QMessageBox()
-        msg.setIcon(QtWidgets.QMessageBox.Information)
+        msg = QMessageBox()
+        msg.setIcon(QMessageBox.Information)
         msg.setText("D-FAST Morphological Impact " + dfastmi.__version__)
         msg.setInformativeText("Copyright (c) 2024 Deltares.")
```

Comment:
Maybe we should also move these strings to something like a resource?
e.g. followup issue.

Reply:
is followup issue

Reply:
Is an issue already created?

Reply:
DFAST-186